testGeogrid_ISCE.py

```diff
@@ -191,6 +190,7 @@ def coregisterLoadMetadataOptical(indir_m, indir_s):
     import struct
     import re
 
+    import isce
```

Comment:
Does excluding this import give you error message? I usually do not import the whole isce library and it worked well.

Reply:
ISCE  does these shenanigans on import:
https://github.com/isce-framework/isce2/blob/main/__init__.py#L41-L44

So, unless  users have done that to their environment manually, you need to `import isce` before trying to import anything inside those paths. Conda install of `isce`, for example, expect you to `import isce` first.